### PR TITLE
Return bundle id in header

### PR DESCRIPTION
**Pull request by Sondergaard**
## Description

Set the bundle id in the header

### CODE REVIEW

File: source/Microsoft.Azure.Functions.Isolated.TestDoubles/MockFunctionDefinition.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.Azure.Functions.Worker;
+
+namespace Microsoft.Azure.Functions.Isolated.TestDoubles
+{
+    public class MockFunctionDefinition : FunctionDefinition
```
Comment (michaelmajgaard) [inline source/Microsoft.Azure.Functions.Isolated.TestDoubles/MockFunctionDefinition.cs:22]: Don't think this file is formatted

File: source/Energinet.DataHub.PostOffice.EntryPoint.MarketOperator/Functions/PeekAggregationsFunction.cs [hunk 2]
```diff
@@ -26,7 +26,8 @@ namespace Energinet.DataHub.PostOffice.EntryPoint.MarketOperator.Functions
 {
     public sealed class PeekAggregationsFunction
     {
-        private const string BundleIdQueryName = "bundleId";
+        public const string BundleIdQueryName = "bundleId";
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.PostOffice.EntryPoint.MarketOperator/Functions/PeekAggregationsFunction.cs:29]: Should we have a common place for these constants?

File: source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekAggregationsAzureFunctionTests.cs [hunk 3]
```diff
@@ -23,6 +23,7 @@
 using Energinet.DataHub.PostOffice.Tests.Common.Auth;
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekAggregationsAzureFunctionTests.cs:23]: Unnecessary using

File: source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekMasterDataAzureFunctionTests.cs [hunk 4]
```diff
@@ -23,6 +23,7 @@
 using Energinet.DataHub.PostOffice.Tests.Common.Auth;
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekMasterDataAzureFunctionTests.cs:23]: Unnecessary using

File: source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekTimeSeriesAzureFunctionTests.cs [hunk 5]
```diff
@@ -23,6 +23,7 @@
 using Energinet.DataHub.PostOffice.Tests.Common.Auth;
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekTimeSeriesAzureFunctionTests.cs:23]: Unnecessary using

File: source/Energinet.DataHub.PostOffice.EntryPoint.MarketOperator/Functions/PeekAggregationsFunction.cs (hunk 2 above)
Comment (Sondergaard) [reply source/Energinet.DataHub.PostOffice.EntryPoint.MarketOperator/Functions/PeekAggregationsFunction.cs:29]: I plan to extract the const to a single file in the next PR

File: source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekAzureFunctionTests.cs [hunk 6]
```diff
@@ -23,6 +23,7 @@
 using Energinet.DataHub.PostOffice.Tests.Common.Auth;
```
[the remark this answers is not in the document]
Comment (Sondergaard) [reply source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekAzureFunctionTests.cs:23]: Removed

File: source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekAggregationsAzureFunctionTests.cs (hunk 3 above)
Comment (Sondergaard) [reply source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/PeekAggregationsAzureFunctionTests.cs:23]: Removed

File: source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/BundledIdTests.cs [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Energinet.DataHub.PostOffice.Application.Commands;
+using Energinet.DataHub.PostOffice.Common.Auth;
```
[the remark this answers is not in the document]
Comment (Sondergaard) [reply source/Energinet.DataHub.PostOffice.Tests/Hosts/MarketOperator/BundledIdTests.cs:20]: Removed

**Review by michaelmajgaard — APPROVED** [review]

**Review by xnetsrak — APPROVED** [review]

**Pull request merged**